### DISCUSSION TITLE
add ggntw.com

### DISCUSSION THREAD

**Discussion by rocketduda** [iMAboud/iMSteam#6] [Ideas]
I suggest adding **ggntw.com** they are a small company, but they are well established, they have a few games, but mostly simulators, it may be an idea for you to create an add-on for direct installation of mods for games from styme, they have an **analog** workshopdownloader

**Comment by iMAboud** [comment]
This site requires user log-in to use its search!

The method I'm using is, to go to a site, search for something and check the link for that quarry, this will give me the link which should be used in this script.

You can do this yourself with almost any site by following these steps:

Go to any site e.g: https://rutracker.org/

Do a search, e.g: The Witcher

Notice the link changes to: https://rutracker.org/forum/tracker.php?nm=the+witcher

Remove "the+witcher" to be like this: https://rutracker.org/forum/tracker.php?nm=

Then add it to the user-script like this:

    const rutracker= `https://rutracker.org/forum/tracker.php?nm=${formattedGameName}`;
    createButton(rutracker, "RuTracker", "Search on RuTracker", "URL_FOR_ICON.PNG");

(Make sure to use this exact format, and add its name in lower case for IDs e.g (rutacker), also add a png icon link).
